contracts/VALORTimedMint.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "@openzeppelin/contracts/access/Ownable.sol"; //not 100% necessary I think, just a modifier would suffice as well
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract VALORTimedMint is ERC20, Ownable {
```

Comment:
I made the ERC20TimedMint with the idea that you would do the work there. Can you merge those two files?